test/transform/classTest.js

```diff
@@ -229,6 +230,8 @@ describe('Classes', () => {
     );
   });
 
+  classInheritance();
```

Comment:
I think it's fine to just have completely separate tests for the inheritance part of class transform. No need to struggle with exporting the tests as a function to pull them inside class test.